tests/__snapshots__/dist.test.js.snap

```diff
@@ -7,37 +7,253 @@ Array [
+  "./build/static/js/59*chunk*map",
+  "./build/static/js/59*chunk.js",
+  "./build/static/js/6*chunk*map",
+  "./build/static/js/6*chunk.js",
+  "./build/static/js/60*chunk*map",
+  "./build/static/js/60*chunk.js",
+  "./build/static/js/61*chunk*map",
+  "./build/static/js/61*chunk.js",
+  "./build/static/js/62*chunk*map",
+  "./build/static/js/62*chunk.js",
+  "./build/static/js/63*chunk*map",
+  "./build/static/js/63*chunk.js",
+  "./build/static/js/64*chunk*map",
+  "./build/static/js/64*chunk.js",
+  "./build/static/js/65*chunk*map",
+  "./build/static/js/65*chunk.js",
+  "./build/static/js/66*chunk*map",
+  "./build/static/js/66*chunk.js",
+  "./build/static/js/67*chunk*map",
+  "./build/static/js/67*chunk.js",
+  "./build/static/js/68*chunk*map",
+  "./build/static/js/68*chunk.js",
+  "./build/static/js/69*chunk*map",
+  "./build/static/js/69*chunk.js",
   "./build/static/js/7*chunk*map",
   "./build/static/js/7*chunk.js",
+  "./build/static/js/70*chunk*map",
+  "./build/static/js/70*chunk.js",
+  "./build/static/js/71*chunk*map",
+  "./build/static/js/71*chunk.js",
+  "./build/static/js/72*chunk*map",
+  "./build/static/js/72*chunk.js",
+  "./build/static/js/73*chunk*map",
+  "./build/static/js/73*chunk.js",
+  "./build/static/js/74*chunk*map",
+  "./build/static/js/74*chunk.js",
+  "./build/static/js/75*chunk*map",
+  "./build/static/js/75*chunk.js",
+  "./build/static/js/76*chunk*map",
+  "./build/static/js/76*chunk.js",
+  "./build/static/js/77*chunk*map",
+  "./build/static/js/77*chunk.js",
+  "./build/static/js/78*chunk*map",
+  "./build/static/js/78*chunk.js",
+  "./build/static/js/79*chunk*map",
+  "./build/static/js/79*chunk.js",
   "./build/static/js/8*chunk*map",
   "./build/static/js/8*chunk.js",
-  "./build/static/js/9*chunk*map",
-  "./build/static/js/9*chunk.js",
+  "./build/static/js/80*chunk*map",
+  "./build/static/js/80*chunk.js",
+  "./build/static/js/81*chunk*map",
+  "./build/static/js/81*chunk.js",
+  "./build/static/js/82*chunk*map",
+  "./build/static/js/82*chunk.js",
+  "./build/static/js/83*chunk*map",
+  "./build/static/js/83*chunk.js",
+  "./build/static/js/84*chunk*map",
+  "./build/static/js/84*chunk.js",
+  "./build/static/js/85*chunk*map",
+  "./build/static/js/85*chunk.js",
+  "./build/static/js/86*chunk*map",
+  "./build/static/js/86*chunk.js",
+  "./build/static/js/87*chunk*map",
+  "./build/static/js/87*chunk.js",
+  "./build/static/js/88*chunk*map",
+  "./build/static/js/88*chunk.js",
+  "./build/static/js/89*chunk*map",
+  "./build/static/js/89*chunk.js",
+  "./build/static/js/90*chunk*map",
+  "./build/static/js/90*chunk.js",
+  "./build/static/js/91*chunk*map",
+  "./build/static/js/91*chunk.js",
+  "./build/static/js/92*chunk*map",
+  "./build/static/js/92*chunk.js",
+  "./build/static/js/93*chunk*map",
+  "./build/static/js/93*chunk.js",
+  "./build/static/js/94*chunk*map",
+  "./build/static/js/94*chunk.js",
+  "./build/static/js/95*chunk*map",
+  "./build/static/js/95*chunk.js",
```

Comment:
required
looks like we solved not including snapshots, but this still appears to be including unit tests?

Reply:
wp exclude corrected it